java-checks-test-sources/src/main/java/checks/DefaultEncodingUsageCheck.java

```diff
@@ -49,11 +51,24 @@ void myMethod(byte[] bytes, java.io.File file, OutputStream outputStream) {
     new Formatter(outputStream); // Noncompliant
     new Scanner(file); // Noncompliant
     new java.util.Scanner(new java.io.FileInputStream("")); // Noncompliant
-    FileReader reader = null; // Noncompliant
-    FileReader reader3 = new FileReader(""); // Noncompliant
-    java.io.Reader reader2 = fileReader(); // Noncompliant [[sc=30;ec=40]]
-    FileWriter writer = null; // Noncompliant
-    java.io.Writer writer2 = fileWriter(); // Noncompliant
+    FileReader reader1 = null; // Compliant
+    FileReader reader2 = new FileReader(file); // Noncompliant
```

Comment:
It will be nice to check issue location and the message in some places and it will be nice to test message "Replace this "null" with actual charset."